riotfile.py

```diff
@@ -150,7 +150,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             venvs=[
                 Venv(
                     name="codespell",
-                    command="codespell ddtrace/ tests/",
+                    command="codespell --exclude-file=ddtrace/appsec/include/ddwaf.h ddtrace/ tests/",
```

Comment:
Uness I completely misunderstood the codespell docs (https://pypi.org/project/codespell/) I think we want to use `--skip` instead of `--exclude-file`?

Reply:
Indeed, we should use `--skip`